src/model.cpp

```diff
@@ -691,16 +742,29 @@ void flattenComponent(const ComponentEntityPtr &parent, const ComponentPtr &comp
             importedComponentCopy->addComponent(component->component(i));
         }
 
-        // Temporarily add component to new model to find units used.
+        // Temporarily add component to new model to find units used by the variables
```

Comment:
Missing full stop at the end of the sentence.

Reply:
Done